src/main/java/edu/hm/hafner/analysis/parser/GhsMultiParser.java

```diff
@@ -39,14 +39,24 @@ public GhsMultiParser() {
     @Override
     protected Optional<Issue> createIssue(final Matcher matcher, final LookaheadStream lookahead,
             final IssueBuilder builder) {
-        String type = StringUtils.capitalize(matcher.group(3));
-        String messageStart = matcher.group(5);
+        String type = StringUtils.capitalize(matcher.group(4));
+        String messageStart = matcher.group(6);
 
         String message = extractMessage(messageStart, lookahead);
 
+        if (StringUtils.isNotBlank(matcher.group(3))) {
```

Comment:
It would be simpler if you set the common builder properties before the `if` (DRY). 

Moreover, I would suggest to replace the indices with symbols, then the regexp and the associated code is easier to follow. A good example can be found here: https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/EclipseParser.java#L24